src/WiFiConnectionManager.h

```diff
@@ -52,19 +52,19 @@ class WiFiConnectionManager : public ConnectionManager {
 
     void changeConnectionState(NetworkConnectionState _newState);
 
-    const int CHECK_INTERVAL_IDLE = 100;
-    const int CHECK_INTERVAL_INIT = 100;
-    const int CHECK_INTERVAL_CONNECTING = 500;
-    const int CHECK_INTERVAL_CONNECTED = 10000;
-    const int CHECK_INTERVAL_RETRYING = 5000;
-    const int CHECK_INTERVAL_DISCONNECTED = 1000;
-    const int CHECK_INTERVAL_ERROR = 500;
+    const unsigned long CHECK_INTERVAL_IDLE = 100;
```

Comment:
since these are just intervals of milliseconds never exceeding the 60 seconds, aren't we just fine wth `uint16_t`?
this is valid for all of the ones coming afterwards

Reply:
The return type of `millis` is of type `unsigned long` - if we reduce those constants to `uint16_t` we would need to cast at the point of their usage.